File: AndroidBasics/notes/src/main/java/pan/alexander/notes/data/database/AppDatabase.java
package pan.alexander.notes.data.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import pan.alexander.notes.domain.entities.Note;

@Database(entities = {Note.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract NotesDao getNotesDao();
}

File: Filmrevealer/app/src/main/java/pan/alexander/filmrevealer/data/database/AppDatabase.kt
package pan.alexander.filmrevealer.data.database

import androidx.room.Database
import androidx.room.RoomDatabase
import pan.alexander.filmrevealer.domain.entities.Film
import pan.alexander.filmrevealer.domain.entities.FilmDetails

@Database(entities = [Film::class, FilmDetails::class], version = 8)
abstract class AppDatabase : RoomDatabase() {
    abstract fun filmDao(): FilmDao
    abstract fun filmDetailsDao(): FilmDetailsDao
}
